frontend/src/reducers/users.ts

```diff
@@ -1,12 +1,28 @@
 import { Action, ActionTypes, UsersData } from '../actions';
 
+const followUserFunc = (state: UsersData, action: Action) => {
+  let users = state.data || [];
+  let to_follow = users.find((u) => u.id === action.payload);
```

Comment:
use readable variables
```suggestion
  let to_follow = users.find((user) => user.id === action.payload);
```

Reply:
done